setup.py

```diff
@@ -36,7 +36,7 @@
         'pycodestyle>=2.8.0',
         'netCDF4>=1.5.3',
         'matplotlib>=3.5.2',
-        'cartopy>=0.20.2',
+        'cartopy>=0.18.0',
```

Comment:
ugh we have to revert again? Doesn't this reversion fix all of the proj issues?

Reply:
I just figured I would set the minimum to this version to eliminate the issues for people who have that old version of proj. But I think we should still have the requirements GitHub file that fixes the versions. That way as packages get updated we don't see the breakage where new versions of packages don't realize that cartopy is tethered to some old thing. That was what caused the break to CI in Eva and EMCpy